### PR TITLE
Remove unnecessary Ubuntu-specific guides

### PR DESCRIPTION
**Pull request by QuintenQVD0**
- Remove he ubuntu2004 and ubuntu1804 files so the search function does not list them anymore, they were already removed from the sidebar
- Fix a forgoten reference to Ubuntu 18.04 been supported on the wings install page
- Remove in the add_node node file the reference to the Ubuntu guide as stock is now Ubuntu and point to the debian guide as the debian9 guide does not exist anymore

### CODE REVIEW

File: wings/1.0/installing.md [hunk 1]
```diff
@@ -18,8 +18,7 @@ only the versions listed below.
 
 | Operating System | Version |     Supported      | Notes                                                       |
 |------------------|---------|:------------------:|-------------------------------------------------------------|
-| **Ubuntu**       | 18.04   | :white_check_mark: | Documentation written assuming Ubuntu 18.04 as the base OS. |
-|                  | 20.04   | :white_check_mark: |                                                             |
+| **Ubuntu**       | 20.04   | :white_check_mark: | Documentation written assuming Ubuntu 20.04 as the base OS. |
 |                  | 22.04   | :white_check_mark: |                                                             |
```
Comment (PadowYT2) [inline wings/1.0/installing.md:22]:
```
| **Ubuntu**       | 20.04   | :white_check_mark: |                                                             |
|                  | 22.04   | :white_check_mark: | Documentation written assuming Ubuntu 20.04 as the base OS. |
```

Comment (QuintenQVD0) [reply wings/1.0/installing.md:22]: ?

Comment (PadowYT2) [reply wings/1.0/installing.md:22]: wait my bad, i thought it should be at the bottom as every other note

**Comment by danny6167** [comment]
Hey @matthewpi this PR mixes in a few changes together, but they are all valid.
 Do you want to merge this anyway as is, or do you want it split up?

**Review by matthewpi — APPROVED** [review]

**Pull request merged**